testing.md

```diff
@@ -0,0 +1,71 @@
+# Testing
+
+## Why write tests?
+
+We write automated tests to allow us to quickly demonstrate that our software meeting business requirements. They're one of a few tools we use to increase our confidence in our work prior to deployment.
```

Comment:
I disagree with this statement, we don't write tests to prove our code does what we say it does. We write tests to enable other developers to contribute to our codebases without introducing regressions, in other words "we write tests to stop other people breaking our stuff". If we ever decide to write tests to prove developers can develop, we're basically saying we don't trust people to do their job.